boards/ti/lp_mspm0g3507/lp_mspm0g3507_mspm0g3507.dts

```diff
@@ -0,0 +1,81 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/dts-v1/;
+
+#include <ti/mspm0g1x0x_g3x0x/mspm0g3507.dtsi>
+#include <ti/mspm0g1x0x_g3x0x/mspm0g350x-pinctrl.dtsi>
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "TI LP_MSPM0G3507/MSPM0G3507";
+	compatible = "ti,mspm0g3507";
+
+	aliases {
+		led0 = &led0;
+		led1 = &led1;
+		sw0 = &btn0;
+		sw1 = &btn1;
+	};
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			gpios = <&gpiob 22 GPIO_ACTIVE_HIGH>;
+			label = "Blue LED";
+		};
+		led1: led_1 {
+			gpios = <&gpiob 26 GPIO_ACTIVE_HIGH>;
+			label = "Red LED";
+		};
+		led2: led_2 {
+			gpios = <&gpiob 27 GPIO_ACTIVE_HIGH>;
+			label = "Green LED";
+		};
```

Comment:
```
git diff boards/ti/lp_mspm0g3507/lp_mspm0g3507_mspm0g3507.dts
diff --git a/boards/ti/lp_mspm0g3507/lp_mspm0g3507_mspm0g3507.dts b/boards/ti/lp_mspm0g3507/lp_mspm0g3507_mspm0g3507.dts
index 166262a7371..cbf8e07b25c 100644
--- a/boards/ti/lp_mspm0g3507/lp_mspm0g3507_mspm0g3507.dts
+++ b/boards/ti/lp_mspm0g3507/lp_mspm0g3507_mspm0g3507.dts
@@ -38,6 +38,10 @@
                        gpios = <&gpiob 27 GPIO_ACTIVE_HIGH>;
                        label = "Green LED";
                };
+               led3: led_3 {
+                       gpios = <&gpioa 0 GPIO_ACTIVE_LOW>;
+                       label = "Red LED";
+               };
```
there is also another RED LED which can be added.

Reply:
You got it